SU-SOE/stanford_paragraph_types
# SOE-3196: Adding js changes for featured paragraph

# READY FOR REVIEW


# Summary
- SOE Featured paragraph type

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for the release.

# Steps to Test
- checkout branch `7.x-1.x-SOE-3196`
- `drush fr stanford_paragraph_types -y --force`
- go to `block/add/stanford-featured-block` and create a featured block.
- play around with the image, video, and image with video make sure things look good.

![screen shot 2018-09-13 at 10 54 37 am](https://user-images.githubusercontent.com/571421/45500433-40587c00-b744-11e8-9fbc-7c51b9927d14.png)
![screen shot 2018-09-13 at 10 54 45 am](https://user-images.githubusercontent.com/571421/45500432-40587c00-b744-11e8-9ef6-fb2aa54bb76d.png)

Examples of the featured blocks:

Image only:
![screen shot 2018-09-13 at 2 55 53 pm](https://user-images.githubusercontent.com/571421/45512402-3f841200-b765-11e8-8ad3-d602b59d1d8e.png)

Image and Video
![screen shot 2018-09-13 at 2 55 47 pm](https://user-images.githubusercontent.com/571421/45512422-4ca10100-b765-11e8-9a08-82b4812cbb22.png)

Video Only
![screen shot 2018-09-13 at 2 56 00 pm](https://user-images.githubusercontent.com/571421/45512448-56c2ff80-b765-11e8-9ac0-a972215e5c78.png)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3196

## Related PRs
Requires: https://github.com/SU-SOE/stanford_bean_types/pull/13

## More Information

## Folks to notify

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
